sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3396,13 +3397,24 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
 
   test("SPARK-30870: Column pruning shouldn't alias a nested column if it means the whole " +
     "structure") {
```

Comment:
nit: this is not related to this pr though, for simplicity, can we pack the test title in a single line like `test("SPARK-30870: Column pruning shouldn't alias a nested column for the whole structure)`?

Reply:
ok.

Reply:
Thanks!